src/mudlet-lua/lua/Other.lua

```diff
@@ -184,8 +184,8 @@ function appendScript(name, luaCode, pos)
   pos = pos or 1
   assert(type(name) == "string", "appendScript: bad argument #1 type (script name as string expected, got "..type(name).."!)")
   assert(type(name) == "string", "appendScript: bad argument #2 type (lua code as string expected, got "..type(luaCode).."!)")
-  if not getScript(name, pos) then
```

Comment:
:thinking: Surely it is more efficient to capture the output from the call to `getScript(name, pos)` and test the (first) returned value rather than calling it twice? That would be the case I suspect for C but I dunno whether the same sort of thing applies in Lua?

Reply:
Yes, this is probably not needed anymore. It was needed before when the error message was returning nil.